perf/src/packet.rs

```diff
@@ -153,4 +153,13 @@ mod tests {
         assert_eq!(rv[0].packets.len(), NUM_PACKETS);
         assert_eq!(rv[1].packets.len(), 1);
     }
+
+    #[test]
+    fn test_to_packets_pinning() {
```

Comment:
This needs to run with Cuda. Do these tests automatically run with stable perf?

Reply:
Yea, it should.

Reply:
```
sakridge@sagan:~/src/silk/perf$ RUST_LOG=solana=info TEST_PERF_LIBS_CUDA=1 cargo test test_to_packets_pinning
   Compiling solana-perf v0.21.0 (/home/sakridge/src/silk/perf)
    Finished dev [unoptimized + debuginfo] target(s) in 6.86s
     Running /home/sakridge/src/silk/target/debug/deps/solana_perf-23ff4a7fae9b348c

running 1 test
[2019-11-20T17:47:06.099170239Z INFO  solana_perf::perf_libs] perf-libs found at "/home/sakridge/src/silk/target/debug/deps/perf-libs"
[2019-11-20T17:47:06.099500359Z INFO  solana_perf::perf_libs] CUDA installation found at "/usr/local/cuda-10.1"
[2019-11-20T17:47:06.099644819Z INFO  solana_perf::perf_libs] LD_LIBRARY_PATH set to "/usr/local/cuda-10.1/lib64:/home/sakridge/src/silk/target/debug/build/backtrace-sys-7d6b334440e072e
2/out:/home/sakridge/src/silk/target/debug/build/clear_on_drop-ea7efd35d5f566a0/out:/home/sakridge/src/silk/target/debug/build/ring-b6890406237bdabd/out:/home/sakridge/src/silk/target/d
ebug/build/sys-info-e68ffb6827694568/out:/home/sakridge/src/silk/target/debug/deps:/home/sakridge/src/silk/target/debug:/home/sakridge/.rustup/toolchains/stable-x86_64-unknown-linux-gnu
/lib/rustlib/x86_64-unknown-linux-gnu/lib:/home/sakridge/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib:/usr/local/cuda/lib64:/home/sakridge/src/silk/target/perf-libs/cuda-10.1:
/usr/local/cuda/lib64:/usr/local/cuda/lib64:/home/sakridge/src/silk/target/perf-libs/cuda-10.1:/usr/local/cuda/lib64:"
[2019-11-20T17:47:06.099820253Z INFO  solana_perf::perf_libs] Loading "/home/sakridge/src/silk/target/debug/deps/perf-libs/cuda-10.1/libcuda-crypt.so"
[2019-11-20T17:47:06.100327709Z INFO  solana_perf::cuda_runtime] host_register 0x7f4bb8000f50
[2019-11-20T17:47:07.659546997Z INFO  solana_perf::cuda_runtime] host_register 0x7f4bb91641f0
```

Reply:
Yeah just wanted to make sure it runs with stable-perf and not stable

Reply:
Yea. actually I think we need to add that. I can do that in a follow-up PR. Just core and ledger are tested now.